gpii/node_modules/gpii-oauth2/gpii-oauth2-datastore/test/DbDataStoreTests.js

```diff
@@ -0,0 +1,1190 @@
+/**
+GPII DB Data Store Tests
+
+Copyright 2016 OCAD University
+
+Licensed under the New BSD license. You may not use this file except in
+compliance with this License.
+
+The research leading to these results has received funding from the European Union's
+Seventh Framework Programme (FP7/2007-2013) under grant agreement no. 289016.
+
+You may obtain a copy of the License at
+https://github.com/GPII/universal/blob/master/LICENSE.txt
+*/
+
+"use strict";
+
+var fluid = require("infusion"),
+    gpii = fluid.registerNamespace("gpii");
+
+require("gpii-pouchdb");
+gpii.pouch.loadTestingSupport();
+
+require("gpii-oauth2");
+require("./DbDataStoreTestsUtils.js");
+
+fluid.logObjectRenderChars = 4096;
+
+fluid.defaults("gpii.tests.dbDataStore.findUserById", {
```

Comment:
Where did all of these tests come from? Do they reflect holes in the coverage of our existing DataStoreTests or do they overlap with them?